4_arm.ipynb

```diff
@@ -400,33 +715,73 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Expand \"TF\" to display \"map\" and \"hand_palm_link\" in \"Frames\".\n",
+    "\n",
+    "Confirm that the end effector is in the target position and orientation.\n",
+    "\n",
+    "![title](./imgs/4_map_ee_tf.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
     "    <b>課題</b>\n",
     "    <p>エンドエフェクタの目標位置姿勢を変えて、ロボットを動かしてみましょう。</p>\n",
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Task</b>\n",
+    "    <p>Move the robot by changing the target position of the end effector.</p>\n",
+    "</div>"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# 自分で考えてみましょう。この下に入力できます。\n"
+    "# 自分で考えてみましょう。この下に入力できます。 Write your code here\n"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "# レゴブロックを拾ってみる"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Pick up a Lego block"
```

Comment:
If you are going to only pick up exactly one Lego block, then use this:

    "# Pick up a Lego block"

If you will pick up multiple Lego blocks, then use this:

    "# Pick up Lego blocks"

Reply:
Picking up single object is enough in this notebook (multiple objects is much advanced).